src/apps/testapps/testVertexExhaustive.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020-2021 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file
+ * @brief tests H3 vertex functions.
+ *
+ *  usage: `testVertex`
```

Comment:
nit: remove usage?

Reply:
Will remove. We're not particularly consistent here, lots of tests have this. Will do a pass to remove everywhere in a later PR.